docs/ci-integration/ci-integration.md

```diff
@@ -0,0 +1,118 @@
+# Earthly in CI
+
+Continuous Integration systems are as varied as the companies that use them. Fortunately, Earthly is flexible enough to fit into any of them.
```

Comment:
I worry that this sounds a little too sales-like, and over-promises by using the word `any`. e.g. will earthly still work if the CI system doesn't permit running under privileged mode?

Reply:
What would you use for an introduction?

Reply:
Maybe something like this, which sets up the next section which discusses dependencies/requirements?

```suggestion
Continuous Integration systems are as varied as the companies that use them; however, Earthly aims to fit into any of them wherever possible.
```

Reply:
So I took a whack at removing some of this sales-ification. I don't think we are overpromising with this new language, but we do need an air of confidence here...